hera_cal/io.py

```diff
@@ -626,6 +635,92 @@ def load_flags(flagfile, filetype='h5', return_meta=False):
         return flags
 
 
+def get_file_lst_range(filepaths, filetype='uvh5', add_int_buffer=False):
+    """
+    Get a file's start, stop and dlst in radians.
+
+    Start and stop are bin center.
+
+    Miriad standard is bin start, so shift by int_time / 2 is done.
+    UVH5 standard is bin center, so lsts are left untouched.
+
+    Parameters:
+    ------------
+    filepaths : type=list, list of filepaths
+
+    filetype : str, options=['miriad', 'uvh5']
+
+    add_int_buffer : type=bool, if True, extend stop times by an integration duration.
+
+    Output: (file_starts, file_stops, int_times)
+    -------
+    file_starts : file starting point (bin center) in LST [radians]
+
+    file_stops : file ending point (bin center) in LST [radians]
+
+    int_times : integration duration in LST [radians]
+    """
+    _array = True
+    # check filepaths type
+    if isinstance(filepaths, str):
+        _array = False
+        filepaths = [filepaths]
+        if filetype not in ['miriad', 'uvh5']:
+            raise ValueError("filetype {} not recognized".format(filetype))
+
+    # form empty lists
+    file_starts = []
+    file_stops = []
+    int_times = []
+
+    # get Nfiles
+    Nfiles = len(filepaths)
+
+    # iterate over filepaths and extract time info
+    for i, f in enumerate(filepaths):
+        if filetype == 'miriad':
+            uv = aipy.miriad.UV(f)
+            # get integration time
+            int_time = uv['inttime'] * 2 * np.pi / (23.9344699 * 3600.)
```

Comment:
I recommend using `astropy.units.si.sday.in_units(astropy.units.si.s)`. You get a slightly different answer, as it turns out (86164.09164 vs. 86164.09053).